resources/web/wwi/texture_loader.js

```diff
@@ -184,6 +265,43 @@ class _TextureLoaderObject {
 
     if (typeof this.onTextureLoad === 'function')
       this.onTextureLoad();
+
+    this._evaluatePendingData();
+  }
+
+  _setTimeout() {
+    // Set texture loading timeout.
+    // If after some time no new textures are loaded, the hasPendingData variable is automatically
+    // reset to false in order to handle not found textures.
+    // The `this.loadingTextures` dictionary is not reset so that it is still possible to recore late textures.
```

Comment:
What does "recore" mean? I guess it's a typo.

Reply:
yes. I don't remember what I meant, but `load` is fine.